src/main/java/com/spotinst/sdkjava/model/api/aws/managed/instance/ApiTasks.java

```diff
@@ -0,0 +1,109 @@
+package com.spotinst.sdkjava.model.api.aws.managed.instance;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.enums.SchedulingTaskTypeEnum;
+import com.spotinst.sdkjava.model.RecurrenceFrequencyEnum;
```

Comment:
Define this enum class in enums folder and use from there.

Reply:
created RecurrenceFrequencyEnum in Enums folder